src/commands/createFunctionApp/stacks/backupStacks.ts

```diff
@@ -270,63 +270,6 @@ export const backupStacks: string = `{
                 }
               ]
             },
-            {
-              "displayText": ".NET Core 3",
-              "value": "dotnetcore3",
-              "minorVersions": [
-                {
-                  "displayText": ".NET Core 3.1",
-                  "value": "3.1",
-                  "stackSettings": {
-                    "windowsRuntimeSettings": {
-                      "runtimeVersion": "3.1",
-                      "appInsightsSettings": {
-                        "isSupported": true
-                      },
-                      "remoteDebuggingSupported": false,
-                      "gitHubActionSettings": {
-                        "isSupported": true,
-                        "supportedVersion": "3.1.301"
-                      },
-                      "appSettingsDictionary": {
-                        "FUNCTIONS_WORKER_RUNTIME": "dotnet"
-                      },
-                      "siteConfigPropertiesDictionary": {
-                        "use32BitWorkerProcess": true
-                      },
-                      "supportedFunctionsExtensionVersions": [
-                        "~3"
-                      ],
-                      "endOfLifeDate": "2022-12-03T00:00:00Z",
-                      "isDeprecated": true
-                    },
-                    "linuxRuntimeSettings": {
-                      "runtimeVersion": "dotnet|3.1",
-                      "appInsightsSettings": {
-                        "isSupported": true
-                      },
-                      "remoteDebuggingSupported": false,
-                      "gitHubActionSettings": {
-                        "isSupported": true,
-                        "supportedVersion": "3.1.301"
-                      },
-                      "appSettingsDictionary": {
-                        "FUNCTIONS_WORKER_RUNTIME": "dotnet"
-                      },
-                      "siteConfigPropertiesDictionary": {
-                        "use32BitWorkerProcess": false,
-                        "linuxFxVersion": "dotnet|3.1"
-                      },
-                      "supportedFunctionsExtensionVersions": [
-                        "~3"
-                      ],
-                      "endOfLifeDate": "2022-12-03T00:00:00Z",
-                      "isDeprecated": true
-                    }
-                  }
-                }
-              ]
-            },
             {
               "displayText": ".NET Core 2",
```

Comment:
We should take .NET Core 2 out as well.

Reply:
I thought so too, but the Create Function App options are still showing .NET Core 2 even with the `removeDeprecatedStacks` option set to true.

![image](https://user-images.githubusercontent.com/40250218/214688867-cfb07952-0174-4236-b374-03185aebe0c7.png)

Should we just remove it anyway rather than wait for the API to catch up?

Reply:
It might be because of the API version? In any case, we should _absolutely never_ recommend customers use EOL stuff, so I'd say we do our best to hide it regardless of whether the API is...

Reply:
Hmm it looks like the same thing is happening with version `2022-03-01` 🙁

I will filter it out manually.